### PR TITLE
Raylife AP Automated Test - LPS-163191 | master

### PR DESCRIPTION
**Pull request by itstali**
https://issues.liferay.com/browse/LPS-163191

### CODE REVIEW

**Comment by itstali** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:test:site-initializer-raylife-ap:nocompile

File: modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPUnderwritingTrigger.testcase [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+				relationToContact = "Spouse");
+
+			RaylifeApplicationModal.selectOptionInDropdownByName(
+				field = "accident-citation",
+				text = "Accident - Not at fault");
+
+			Button.click(button = "Next");
+		}
+
+		task ("Then check if the user could add an Accident Citation") {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPUnderwritingTrigger.testcase:99]: Please check the task block and final assertion. The task block is saying that the Citation can be added, but the assertion is checking that something is not there. 

Please add the following task blocks / steps:
1. Complete the application
2. Navigate to the application
3. Assert that the citation was added/visible.

**Review by nikki-pru — CHANGES_REQUESTED** [review]
Hi Talitha!

Please review changes. I added some notes for the final assertion. Let's chat if you have questions about the added steps.

**Comment by itstali** [comment]
ci:test:sf

**Comment by renato-araujo-liferay** [comment]
ci:test:site-initializer-raylife-ap:nocompile

**Review by renato-araujo-liferay — APPROVED** [review]
Reviewed and approved

**Review by renato-araujo-liferay — APPROVED** [review]

**Comment by renato-araujo-liferay** [comment]
ci:forward

**Comment by itstali** [comment]
ci:forward

**Pull request closed without merging**